src/main/java/uk/gov/pay/api/model/CreateCardPaymentRequest.java

```diff
@@ -136,6 +137,8 @@ public String getReturnUrl() {
     }
 
     @JsonProperty("metadata")
+    @ApiModelProperty(value = "Additional metadata - up to 10 key-value pairs - on the payment. Each key must be between 1 and 30 characters long. Values must be no greater than 50 characters long and be of types String, Boolean or Number.",
+            dataType = "java.util.Map", example = "{'ledger_code':'123', 'reconciled', true}")
```

Comment:
JSON ~keys~ names have to be in double quotes and I think that comma after “reconciled” should be a boolean.